src/uhttp.c

```diff
@@ -860,7 +861,7 @@ static int construct_http_headers(HTTP_HEADERS_HANDLE http_header, size_t conten
         if (result == 0)
         {
             /* Codes_SRS_UHTTP_07_015: [uhttp_client_execute_request shall add the Content-Length to the request if the contentLength is > 0] */
-            size_t fmtLen = strlen(HTTP_CONTENT_LEN)+strlen(HTTP_CRLF_VALUE)+8;
+            size_t fmtLen = strlen(HTTP_CONTENT_LEN)+HTTP_CRLF_LEN+8;
```

Comment:
>strlen(HTTP_CONTENT_LEN)+HTTP_CRLF_LEN+8 [](start = 28, length = 40)

same here

Reply:
Thanks @danewalton - went through the whole file and tried to fix them all.

Since you're pointing these out, we should have a coding style document so that the code looks the same. 
Looking in our main repo, I can't find one. Is there one available that I've missed?

Reply:
i personally don't know of any for the c sdk. the rtos team did but i'm not aware for the c sdk.